src/functions/module_lookup.ts

```diff
@@ -153,7 +156,7 @@ export const HLOOKUP = {
   args: [
     arg("search_key (any)", _t("The value to search for. For example, 42, 'Cats', or I24.")),
     arg(
-      "range (range)",
+      "range (any, range)",
```

Comment:
Do they also accept error values ?
It doesn't matter in 17.0, but it does in the version in which we did the "early return" optimization
```suggestion
      "range (string,number,boolean,range)",
```